remote/remote_test.go

```diff
@@ -20,7 +20,14 @@ import (
 	h "github.com/buildpacks/imgutil/testhelpers"
 )
 
-var dockerRegistry, readonlyDockerRegistry *h.DockerRegistry
+var dockerRegistry, readonlyDockerRegistry, customRegistry *h.DockerRegistry
+
+const (
+	readWriteImage      = "image-readable-writable"
+	readNoWriteImage    = "image-readable"
```

Comment:
```suggestion
	onlyReadImage    = "image-readable"
```

Reply:
Added in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/87d5ba9fc355d30923e2cf672f08e02c9de908fd)